UI/src/index.html

```diff
@@ -2,13 +2,57 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>LitPaths</title>
+  <title>Litpaths - Find street lit places to walk and exercise</title>
+    <meta
+      name="title"
+      content="Litpaths - Find street lit places to walk and exercise"
+    />
+    <meta
+      name="description"
+      content="Litpaths allows you to find paths and routes that are lit by street lamps, should you want to walk or exercise and stay away from the dark."
```

Comment:
Should we add that "Scotland"? So that people know that currently we have Scottish data so far

Reply:
I'm inclined to not make it specific incase the coverage grows. And if other services like Google cache this then we're stuck with it.

The caching might not be that bad but I'd need to investigate

Reply:
Yes, it makes sense then!